doc/nrf/releases/release-notes-2.1.0.rst

```diff
@@ -0,0 +1,1030 @@
+
+* :ref:`bt_fast_pair_readme` service:
+
+  * Added:
+
+    * A SHA-256 hash check to ensure the Fast Pair provisioning data integrity.
+    * Unit test for the storage module.
+
+  * Updated API to allow setting the flag for the hide UI indication in the Fast Pair not discoverable advertising data.
+
+* :ref:`bt_enocean_readme` library
+
+  * Added callback :c:member:`decommissioned` to :c:struct:`bt_enocean_callbacks` when EnOcean switch is decommissioned.
+
+* :ref:`bt_mesh`:
+
+  * Added:
+
+    * Use of decommissioned callback in :ref:`bt_mesh_silvair_enocean_srv_readme` when EnOcean switch is decommissioned.
+    * :ref:`emds_readme` support to:
+
+      *  :ref:`bt_mesh_plvl_srv_readme`
+      *  :ref:`bt_mesh_light_hue_srv_readme`
+      *  :ref:`bt_mesh_light_sat_srv_readme`
+      *  :ref:`bt_mesh_light_temp_srv_readme`
+      *  :ref:`bt_mesh_light_xyl_srv_readme`
+      *  :ref:`bt_mesh_lightness_srv_readme`
+      * Replay protection list (RPL).
+
+  * Updated the ``bt_mesh_sensor_ch_str_real`` function by replacing it with the :c:func:`bt_mesh_sensor_ch_str` function, which was previously a macro.
+
+Bootloader libraries
+--------------------
+
+* :ref:`lib_dfu_target` library:
+
+   * Moved the :c:func:`dfu_ctx_mcuboot_set_b1_file` function to :ref:`lib_fota_download` and renamed it to :c:func:`fota_download_parse_dual_resource_locator`.
+
+Modem libraries
+---------------
+
+* :ref:`lte_lc_readme` library:
+
+  * Fixed an issue that caused stack corruption in the :c:func:`lte_lc_nw_reg_status_get` function.
+
+* :ref:`at_monitor_readme` library:
+
+  * Reworked what previously were macros to :c:func:`at_monitor_pause` and :c:func:`at_monitor_resume` functions.
+    These new functions and take a pointer to the AT monitor entry.
+
+* :ref:`modem_key_mgmt` library:
+
+  * Fixed an issue that would cause the library to assert on an unhandled CME error when the AT command failed to be sent.
+
+* :ref:`at_cmd_parser_readme` library:
+
+  * Fixed an issue that would cause AT command responses like ``+CNCEC_EMM`` with underscore to be filtered out.
+
+* :ref:`pdn_readme` library:
+
+  * Added:
+
+    * Support for setting multiple event callbacks for the default PDP context.
+    * The :c:func:`pdn_default_ctx_cb_dereg` function to deregister a callback for the default PDP context.
+    * The :c:func:`pdn_esm_strerror` function to retrieve a textual description of an ESM error reason.
+      The function is compiled when :kconfig:option:`CONFIG_PDN_ESM_STRERROR` Kconfig option is enabled.
+
+  * Updated:
+
+    * The :c:func:`pdn_default_callback_set` function name to :c:func:`pdn_default_ctx_cb_reg`.
+    * Automatic subscribtion to ``+CNEC=16`` and ``+CGEREP=1`` if the :ref:`lte_lc_readme` library is used to change the modem's functional mode.
+
+  * Removed the ``CONFIG_PDN_CONTEXTS_MAX`` Kconfig option.
+    The maximum number of PDP contexts is now dynamic.
+
+* :ref:`nrf_modem_lib_readme` library:
```

Comment:
This now reads a bit strange (Modem library integration layer library)
Link should perhaps point to https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrfxlib/nrf_modem/README.html instead of 
https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/libraries/modem/nrf_modem_lib.html

at least for the "headline", and then maybe add a link to the integration layer below?
@lemrey, thoughts?

Reply:
I'll remove "library" after the link. We can't link to nrfxlib here, as these are release notes for the sdk-nrf part, not sdk-nrfxlib part, which has its own release notes.